BigInteger.js

```diff
@@ -1314,6 +1314,7 @@ var bigInt = (function (undefined) {
 
     BigInteger.prototype.toString = function (radix, alphabet) {
         if (radix === undefined) radix = 10;
+        if (radix === 2) return (this.sign ? "-" : "") + this.toArray(2).value.join('');

```

Comment:
what about the `alphabet` argument?

Reply:
Ah, yes i did not consider that. I updated it to only apply the optimization if the alphabet parameter is not provided. That may be a bit hacky now, whatcha think?